lib/active_fedora/indexing.rb

```diff
@@ -87,15 +87,17 @@ def index_config
         # @param [Boolean] softCommit - Do we perform a softCommit when we add the to_solr objects to SolrService. Default true.
         # @param [Boolean] progress_bar - If true output progress bar information. Default false.
         # @param [Boolean] final_commit - If true perform a hard commit to the Solr service at the completion of the batch of updates. Default false.
-        def reindex_everything(batch_size: 50, softCommit: true, progress_bar: false, final_commit: false)
+        def reindex_everything(batch_size: 50, softCommit: true, progress_bar: false, final_commit: false) # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
@jrochkind I think it worked because it turned it off until the end of the file.